openfga/v1/authzmodel.proto

```diff
@@ -52,19 +52,11 @@ message TypeDefinition {
 }
 
 message Relation {
-  string name = 1 [(validate.rules).string = {
```

Comment:
The reason I want to remove this is that if we always use it as a `map[string]*Relation` internally then having to store the name in two places is redundant and, more importantly, error prone. If we have something like
```
"document": {
  Name: "documant"  // misspell
  ...
}
```
that is obviously very bad.